tests/IBTK/curl_01_2d.output

```diff
@@ -0,0 +1 @@
+|e|_oo = 0.39974
```

Comment:
The large error is because of the coarse-fine interface treatment, which is using `CONSERVATIVE_LINEAR_REFINE` to fill ghost values --- so we only expect (and realize) first order accuracy.

Reply:
I checked and we do get first-order convergence in the max norm - for some reason the visualization of the error doesn't include this jump and looks smooth. Must be a visit thing.